src/components/Header.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "./utils/index.scss";
+.head_block {
+  @include theme($theme: $darkBlue);
+  transition:
+    background-color 0.8s,
+    color 0.8s;
+  width: 100%;
+  min-height: 100px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 20px;
+  align-items: center;
+}
+h3 {
+  font-weight: normal;
+  cursor: pointer;
+}
+img {
+  padding-top: 1px;
+  width: 1em;
+  height: 1em;
+  margin-right: 0.5em;
+  filter: invert(100%) brightness(1000%);
```

Comment:
Are you sure you need this filter?

Reply:
yes, because  color: white doesn’t  work

Reply:
This is why you need SVG instead of png.

Reply:
With svg you can use any color you want https://www.youtube.com/watch?v=b4btk12Evgw

Reply:
I tried  a lot ^ but there is an issue  the I can’t  resolve

Reply:
Which one?

Reply:
Uncaught SyntaxError: The requested module '/src/assets/images/half-moon-shape-svgrepo-com.svg?import' does not provide an export named 'ReactComponent' (at Header.jsx:2:10)